# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/shell/fuzz-flags.txt
# Commit: 5c5cf716aa0b
# Full Hash: 5c5cf716aa0b01aecd8702254427825c9de1b8fa
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2023-06-08 03:42:08
# Regressor Bug: 1835876
# File Overlap Count: 1
# Description:
#   Bug 1835876 part 2 - Disable code write protection in content processes. r=nbp
#   
#   We've worked with the security and performance teams to re-evaluate the W^X policy
#   we have in place to mark JIT code memory pages as either writable or executable
#   (but not both).
# ==============================================================================

diff -r 028f981600d7 -r 5c5cf716aa0b js/src/shell/fuzz-flags.txt
--- a/js/src/shell/fuzz-flags.txt	Wed Jun 07 16:34:51 2023 +0000
+++ b/js/src/shell/fuzz-flags.txt	Wed Jun 07 16:34:51 2023 +0000
@@ -50,6 +50,8 @@
 --nursery-bigints=on
 --spectre-mitigations=off
 --spectre-mitigations=on
+--write-protect-code=off
+--write-protect-code=on
 --more-compartments
 --fast-warmup
 --no-jit-backend